ykbh/src/lib.rs

```diff
@@ -0,0 +1,267 @@
+use std::alloc::{alloc, dealloc, Layout};
+use std::convert::{TryFrom, TryInto};
+use ykpack::{
+    self, Constant, ConstantInt, IPlace, Local, LocalDecl, Statement, Terminator, TypeId,
+    UnsignedInt,
+};
+use yktrace::sir::SIR;
+
+const SUSIZE: usize = std::mem::size_of::<usize>();
```

Comment:
I think it's better to just do:

```rust
use std::mem::size_of;
```

and then:

```rust
size_of::<usize>()
```

where necessary. It stops people wondering "what is `SUSIZE`?"

Reply:
This is probably me trying to be too smart. My thinking was that I don't want to have this call each time we need the size of usize. I forgot the compiler will just optimise away the call anyway.

Reply:
Yeah, it's not really a call. [Rust has statically evaluated const expressions which, with my Converge CTMP hat on, look incredibly ad-hoc and verbose. But that's a separate rant.]

Reply:
Fixed in b931d23a549829cbd924ec6f1180f7cf7170bb9e.

Reply:
Fixed in b931d23.